pytest.ini

```diff
@@ -7,8 +7,9 @@ markers =
     network: Mark a test that uses the network.
     style: Set alternate Matplotlib style temporarily.
 
-pep8ignore =
-    * E111 E114 E115 E116 E121 E122 E123 E124 E125 E126 E127 E128 E129 E131 E226 E240 E241 E242 E243 E244 E245 E246 E247 E248 E249 E265 E266 E704 W503
+flake8-max-line-length = 80
```

Comment:
Shouldn't this be left at 79 (the default, i.e. remove this line)?

Reply:
I got this from @cclauss's PR, but I think you're right, so I'm going to remove it.

Reply:
sorry, off-by-one bug on my part.